Makefile

```diff
@@ -60,30 +60,22 @@ build:
 test:
 	./scripts/check-links.sh
 
-.PHONY: travis_push
-travis_push::
+.PHONY: ci_push
+ci_push::
 	$(MAKE) banner
 	$(MAKE) ensure
-ifeq ($(TRAVIS_BRANCH),master)
```

Comment:
I removed these checks because the GitHub actions workflow is set to only run for events against the `master` branches.